app/src/test/scala/org/alephium/explorer/persistence/queries/TransactionQueriesSpec.scala

```diff
@@ -97,7 +99,27 @@ class TransactionQueriesSpec extends AlephiumSpec with ScalaFutures {
     tx.outputs.size is 1 // was 2 in v1.4.1
   }
 
-  trait Fixture extends DatabaseFixture with DBRunner with Generators {
+  it should "insert and update transactions" in new Fixture {
```

Comment:
Does this test still make sense?

Reply:
there might be similar tests elsewhere

Reply:
I actually fixed that test [here](https://github.com/alephium/explorer-backend/pull/152/commits/32a75db4ba5fd7deee73a7135b4a163bfbc9149f) I think it's good to test that that value isn't upserted

Reply:
I was complaining about the naming actually. It should not update transactions now.

Reply:
yep, [this is also fixed](https://github.com/alephium/explorer-backend/pull/152/commits/32a75db4ba5fd7deee73a7135b4a163bfbc9149f#diff-072a7c5f149a9fa5168adadea4ac5a7c0ba8018566faf3307c39ab7b113a4d14R102)